helm/grafana/templates/image-renderer-network-policy.yaml

```diff
@@ -64,10 +64,7 @@ spec:
         - port: {{ .Values.service.port }}
           protocol: TCP
       to:
-        - namespaceSelector:
-            matchLabels:
-              name: {{ template "grafana.namespace" . }}
```

Comment:
Feels strange that the namespaceSelector is gone here but not on other NetworkPolicies

Reply:
I checked, and that's a legit upstream change: issue https://github.com/grafana/helm-charts/issues/795 fixed by https://github.com/clementnuss/helm-charts/commit/82b0677d08a58977719fe5ce20e0ba140d51c3fc